jacodb-ets/src/main/kotlin/org/jacodb/ets/model/EtsSignature.kt

```diff
@@ -65,6 +65,10 @@ data class EtsClassSignature(
             name
         }
     }
+
+    override fun equals(other: Any?): Boolean = (other as? EtsClassSignature)?.let { sig ->
```

Comment:
@zishkaz Please, rewrite it in a common way, as block function, do not squeeze everything into one line. Also, add some TODO/comment, explaining why we need such `equals` implementation, and how/when it will be changed back to the default data class' equals.